CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [[Unreleased]]
 - Add support for the DeliverMax field in Payment transactions
+- Allow empty strings for the purpose of removing fields in DIDSet transaction
```

Comment:
This should be under a `### Fixed` heading

Reply:
fixed in [d0d5d75](https://github.com/XRPLF/xrpl-py/pull/717/commits/d0d5d753e528f5dacbd99e4bf04373d2ba6cb602)